resource-definitions/template-driver/imagepullsecrets/config.yaml

```diff
@@ -0,0 +1,50 @@
+# This Resource Definition pulls credentials for a container image registry from a secret store
+# and creates a Kubernetes Secret of kubernetes.io/dockerconfigjson type
+apiVersion: entity.humanitec.io/v1b1
+kind: Definition
+metadata:
+  id: regcred-config
+entity:
+  driver_type: humanitec/template
+  name: regcred-config
+  type: config
+  criteria:
+  - class: default
+    # This res_id must be used from a referencing Resource Definition to request this config Resource
+    res_id: regcred
+  driver_inputs:
+    # These secret references read the credentials from a secret store
+    secret_refs:
+      password:
+        ref: regcred-password
+        store: FIXME
```

Comment:
Could we add a note on doc above about the FIXME that the users should supply about Secret Store and Server?

Reply:
I added notes above all FIXME values to replace them with the actual values, and also included a hint in the README to do just that. Is this what you had in mind?

Reply:
Nice, yup!